Tasks/PullRequestCodeAnalysis/Tests/L0failsWithoutReportLocation.ts

```diff
@@ -0,0 +1,34 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+
+let taskPath = path.join(__dirname, '..', 'prcatask.js');
+let tmr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+process.env['BUILD_SOURCEBRANCH'] = 'refs/pull/6/master'; //task-lib doesn't support getVariable yet, so we need these
+process.env['BUILD_REPOSITORY_ID'] = '00000000-0000-0000-0000-000000000000';
+process.env['ENDPOINT_AUTH_SYSTEMVSSCONNECTION'] = '{\"scheme\":\"OAuth\", \"parameters\": {\"AccessToken\": \"foobar\"}}';
+process.env['SYSTEM_TEAMFOUNDATIONCOLLECTIONURI'] = 'https://youraccount.visualstudio.com/DefaultCollection';
+
+tmr.setInput('messageLimit', '100');
+
+// provide answers for task mock
+let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
```

Comment:
This object is mostly a one-for-one replacement for response.json except for the aforemention missing getVariable aspect.